src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/spec/response/confirmation/CounterClaimConfirmationText.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user.spec.response.confirmation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.enums.RespondentResponseTypeSpec;
+import uk.gov.hmcts.reform.civil.handler.callback.user.spec.RespondToClaimConfirmationTextSpecGenerator;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+import java.util.Optional;
+
+@Component
+public class CounterClaimConfirmationText implements RespondToClaimConfirmationTextSpecGenerator {
+
+    @Override
+    public Optional<String> generateTextFor(CaseData caseData) {
+        if (!RespondentResponseTypeSpec.COUNTER_CLAIM.equals(caseData.getRespondent1ClaimResponseTypeForSpec())) {
+            return Optional.empty();
+        }
+
+        StringBuilder sb = new StringBuilder();
+        sb.append("<br>You've chosen to counterclaim - this means your defence cannot continue online."
+                      + " Use form N9B to counterclaim, do not create a new claim.");
```

Comment:
if using StringBuilder, use append instead of +. In any case, this is simple enough you can not even use StringBuilder